src/engine/Minus.h

```diff
@@ -43,10 +43,12 @@ class Minus : public Operation {
   uint64_t getSizeEstimateBeforeLimit() override;
 
   // Create a variant of functions that find undefined values in a range. In
-  // case the selected operation (via `left`) is guaranteed to not contain any
-  // undefined values in the join columns, the function will be replaced with
-  // no-op.
-  auto makeUndefRangesChecker(bool left) const;
+  // case the selected operation (via `left`) might contain static values all
+  // values are checked for undef values. If undef values are found an instance
```

Comment:
There is a typo here "might contain static values" is not semantically useful as a sentence.